functionary/ui/templates/base.html

```diff
@@ -131,16 +129,21 @@ <h1 class="title is-1">
                     {% endif %}
                 </aside>
             {% endif %}
-            <main class="column">
-                {% block content %}
-                {% endblock content %}
+            <main class="column is-9 container">
+                <div id="data_div"
+                     hx-get="{{ request.get_full_path }}"
+                     hx-trigger="reloadData from:body"
+                     hx-select="#dataBlock"
+                     hx-swap="innerHTML"
+                     hx-swap-oob="true">
+                    <div id="dataBlock">
+                        {% block content %}
+                        {% endblock content %}
+                    </div>
+                </div>
```

Comment:
For reason's that are not clear, the addition of these divs and the htmx stuff breaks the parameter loading on schedule task creation page.  We'll either need to sort out what the incompatibility is there or come up with another approach to reloading everything after the environment changes.

Reply:
Fixed by disinheriting all those hx- attributes.